### PR TITLE
[Deepthi M]|Trello-365 | Addded functionality for the inward api

### PR DESCRIPTION
**Pull request by deepthi-mantena**
Screenshot 2023-09-27 at 11 32 20 PM

Attaching the screenshot .

### CODE REVIEW

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package org.bahmni.module.hwcinventory.contract;
+
+public class EAushadhaInwardRequest {
+    private String InwardDate ;
+    private String InstituteId;
```
Comment (riyaTw) [inline api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java:5]: can u follow camel case for variable name

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+    public String getInstituteId() {
+        return InstituteId;
+    }
+
+    public void setInstituteId(String instituteId) {
+        InstituteId = instituteId;
+    }
+
+    public EAushadhaInwardRequest() {
```
Comment (riyaTw) [inline api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java]: Is this empty constructor really needed?

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardResponse.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class EAushadhaInwardResponse {
+    @JsonAlias("Sl_No")
+    private int Sl_No;
```
Comment (riyaTw) [inline api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardResponse.java]: Can we follow camel casing here as well in all variables

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardStockReceiptRequest.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+package org.bahmni.module.hwcinventory.contract;
+
+public class EAushadhaInwardStockReceiptRequest {
+    private String inwardDate;
+    private String instituteId;
+
+    public EAushadhaInwardStockReceiptRequest() {
```
Comment (riyaTw) [inline api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardStockReceiptRequest.java]: Can we remove this unnecessary constructor

File: api/src/main/java/org/bahmni/module/hwcinventory/service/impl/EaushadhaServiceImpl.java [hunk 5]
```diff
@@ -48,6 +50,34 @@ public List<EAushadhaResponse> fetchStockDetails(String outwardId) throws Except
...
+
+
+    public List<EAushadhaInwardResponse> fetchInwardStockDetails(String inwardDate, String instituteId) throws Exception {
+        List<EAushadhaInwardResponse> eAushadhaInwardResponses;
+        try {
+
+            String accessToken = System.getenv("EAUSHADHA_ACCESS_TOKEN");
+
+            String eAushadhaUrl = Context.getAdministrationService().getGlobalProperty("eaushadha.api.baseUrl") + "/api/DWInstituteInward";
+
```
Comment (riyaTw) [inline api/src/main/java/org/bahmni/module/hwcinventory/service/impl/EaushadhaServiceImpl.java]: Can we format this and remove extra line spaces

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java (hunk 1 above)
Comment (deepthi-mantena) [reply api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java:5]: This is the request object for calling EAushadha Api and the contract we received had the variable as declared here
Screenshot 2023-09-27 at 10 59 45 PM

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java (hunk 2 above)
Comment (deepthi-mantena) [reply api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardRequest.java]: Removed

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardResponse.java (hunk 3 above)
Comment (deepthi-mantena) [reply api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardResponse.java]: Updated all variables appropriately

File: api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardStockReceiptRequest.java (hunk 4 above)
Comment (deepthi-mantena) [reply api/src/main/java/org/bahmni/module/hwcinventory/contract/EAushadhaInwardStockReceiptRequest.java]: removed

File: api/src/main/java/org/bahmni/module/hwcinventory/service/impl/EaushadhaServiceImpl.java (hunk 5 above)
Comment (deepthi-mantena) [reply api/src/main/java/org/bahmni/module/hwcinventory/service/impl/EaushadhaServiceImpl.java]: Formatted the spaces

**Pull request merged**